src/bitmessageqt/bitmessageui.py

```diff
@@ -1,20 +1,29 @@
 # -*- coding: utf-8 -*-
+# pylint: disable=too-many-locals
+"""
+Form implementation generated from reading ui file 'bitmessageui.ui'
 
-# Form implementation generated from reading ui file 'bitmessageui.ui'
-#
-# Created: Mon Mar 23 22:18:07 2015
-#      by: PyQt4 UI code generator 4.10.4
-#
-# WARNING! All changes made in this file will be lost!
+Created: Mon Mar 23 22:18:07 2015
+     by: PyQt4 UI code generator 4.10.4
+
+WARNING! All changes made in this file will be lost!
+"""
+
+from __future__ import absolute_import
+
+import sys
 
 from PyQt4 import QtCore, QtGui
+
 from bmconfigparser import BMConfigParser
-from foldertree import AddressBookCompleter
-from messageview import MessageView
-from messagecompose import MessageCompose
-import settingsmixin
-from networkstatus import NetworkStatus
-from blacklist import Blacklist
+from . import bitmessage_icons_rc  # pylint: disable=unused-import
+from . import settingsmixin
+from .messageview import MessageView
+from .messagecompose import MessageCompose
+from .networkstatus import NetworkStatus
+from .blacklist import Blacklist
+from .foldertree import AddressBookCompleter
+
```

Comment:
My usual reminder about dot imports here...

Reply:
Made imports absolute, tested both installed and portable methods.